test/base.js

```diff
@@ -42,6 +42,81 @@ test('Should expose a websocket', (t) => {
   })
 })
 
+test('Should fail if custom errorHandler is not a function', (t) => {
+  t.plan(2)
+
+  const fastify = Fastify()
+  t.tearDown(() => fastify.close())
+
+  fastify
+    .register(fastifyWebsocket, { errorHandler: {} })
+    .after(err => t.equal(err.message, 'invalid errorHandler function'))
+
+  fastify.get('/', { websocket: true }, (connection, request) => {
+    t.tearDown(() => connection.destroy())
+  })
+
+  fastify.listen(0, (err) => {
+    t.error(err)
+  })
+})
+
+test('Should run custom errorHandler on wildcard route handler error', (t) => {
+  t.plan(2)
+
+  const fastify = Fastify()
+  t.tearDown(() => fastify.close())
+
+  fastify.register(fastifyWebsocket, {
+    errorHandler: function (error, connection) {
+      t.equal(error.message, 'Fail')
+    }
+  })
+
+  fastify.get('/*', { websocket: true }, (conn, request) => {
+    conn.pipe(conn)
+    t.tearDown(() => conn.destroy())
+    return Promise.reject(new Error('Fail'))
+  })
```

Comment:
Can you please add a normal /* route and see if everything keep working fine?

Reply:
I meant having a test with both `/*` websocket and `/*` normal. I just want to triple-check this as wildcards are very common (for example in fastify-static).